blog/WorkingEfficicientlyWithJUnitInEclipseUpdate.adoc

```diff
@@ -0,0 +1,79 @@
+= Working Efficiently with JUnit in Eclipse - Update

```

Comment:
@maxandersen : @apupier already submitted a PR to get himself added as an author: https://raw.githubusercontent.com/jbosstools/jbosstools-website/master/_data/team/apupier.adoc